### ISSUE TITLE
[Epoch Guardian] Q3 and arcana Q3 especially need particle control, they lag all clients when launched in multiple enemies

### ISSUE THREAD

**Issue by redledcat** [ChalkyBrush/roshpit-bug-tracker#997]

**Comment by redledcat** [comment]
![20190127222115_1](https://user-images.githubusercontent.com/32621322/51805728-e505d100-2281-11e9-87ca-d1a9f90c8eb2.jpg)

**Comment by DeckardCainRPC** [comment]
i offer a creative solution: make it so that having any points in arcana q3 will, as an added bonus, increase the projectile speed of his attacks by like 1500. this solution would not require dicking with the particles or whatever, and would attempt to fix the problem by making his stream of bullets resolve faster, thus getting them off the map quicker and reducing the traffic

**Comment by redledcat** [comment]
that issue really needs attention
![20190128140443_1](https://user-images.githubusercontent.com/32621322/51832581-24760100-2306-11e9-84a5-32640318d41a.jpg)
![20190128135308_1](https://user-images.githubusercontent.com/32621322/51832603-32c41d00-2306-11e9-9282-93a8c47b3cf7.jpg)

**Comment by redledcat** [comment]
It also feels like some kind of sound effect being disgustingly multiplied when Q arcana casted

**Comment by DeckardCainRPC** [comment]
yes, i was in a game with redledcat's HORRENDOUS OD BUILD and it lagged like you would not believe. let me rephrase that: it fucked up the game so badly that i'm surprised it wasn't one of MY disasters

**Comment by Nintai** [comment]
Solution: Reduce Cast range to 1500

**Comment by DeckardCainRPC** [comment]
DON'T YOU DARE

that rocket emoji is there to represent AMERICA'S NUCLEAR ARSENAL

**Comment by DeckardCainRPC** [comment]
WHICH I'D LIKE TO REMIND YOU ALSO HAS GLOBAL CAST RANGE

**Comment by redledcat** [comment]
you'll be surprised, but it can be cast on enemy cluster in 1500 range as well, and nearly destroy your fps meter

**Comment by redledcat** [comment]
it might be E3 interaction

**Comment by redledcat** [comment]
Yeah E3 probably multiplies particles or attacks, so casting Q arcana on group of enemies potentially creates (enemies)^2 attacks, equal (enemies) on each target, capped by E3 value.
If this is true, correct E3 and Q arcana interaction and buff Q3 damage.
Right now it's possible to kill sven in 2-3 shields break, and only by cheesing him from outside of boss area, using swapping glyphs strategy + stargazer, OD can't fight his ads by any way

**Comment by redledcat** [comment]
doesn't exist anymore
